screwdriver.yaml

```diff
@@ -0,0 +1,15 @@
+shared:
+    image: golang
+    environment:
+        GOPATH: /sd/workspace
+
+jobs:
+    main:
+        requires: [~pr, ~commit]
+        steps:
+            - install: go mod download
+            - vet: go vet ./...
+            - gofmt: ret=$(find . -name '*.go' | xargs gofmt -d) && echo "$ret" && test -z "$ret"
```

Comment:
how about more simplify like meta-cli?
https://github.com/screwdriver-cd/meta-cli/blob/9cfbdc987b7c55e6a03559f0d793d1f321124fa4/screwdriver.yaml#L15

Reply:
And there is a difference between makefile's `gofmt` and screwdriver.yaml's `gofmt` command (`-s` option) .

Reply:
I add s option to screwdriver.yaml.